src/main/java/pl/nogacz/snake/application/SaveGame.java

```diff
@@ -0,0 +1,42 @@
+package pl.nogacz.snake.application;
+
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.ObjectOutputStream;
+
+import pl.nogacz.snake.board.BoardInfo;
+
+public class SaveGame {
+
+    BoardInfo BI;
+    String fileName;
+
+    public SaveGame(BoardInfo BI, String fileName){
+
+        this.BI = BI;
+        this.fileName = fileName;
+    }
+
+    public boolean startSave(){
+
+        FileOutputStream fo;
+        ObjectOutputStream out;
+        
+        String path = "src\\main\\SavedGames\\" + fileName;
```

Comment:
Load için path ile ilgili söylediklerim burası için de geçerli.

Reply:
Done.